### PR TITLE
Made components and calculator

### PR DESCRIPTION
**Pull request by emyrue**
- Use react components
- Use react props
- Use css to style

*Note: In calculator.js, a property in eslint is disabled. This is because I used a class, as instructed, where the linter said I should use a function.

### CODE REVIEW

**Review by Kingstalux — CHANGES_REQUESTED** [review]
Hi @emyrue, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f744b797263304e546c4563475764546870342f67697068792e676966

## Highlights
✅ Good pull request description
✅ Professional README file
✅ Class-based components used 👍 

## Required Changes ♻️
- [ ] Your server starts with the following console error. Kindly clone your repo afresh and install npm so that you might resolve the issue 👍 
- Server error:
Annotation 2022-04-19 204828



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by emyrue** [comment]
Npm install is part of the setup process in the readme.

**Comment by Kingstalux** [comment]
> Npm install is part of the setup process in the readme.

Clone your repo afresh from GitHub and try to run it to see if you'll get any errors

**Comment by emyrue** [comment]
I followed the steps in a new directory and I didn't get any errors.

**Comment by Kingstalux** [comment]
> I followed the steps in a new directory and I didn't get any errors.
 
If you've cloned the repo and it runs okay, then submit it for a re-review

**Comment by emyrue** [comment]
Ok, thanks!

File: src/App.js [hunk 1]
```diff
@@ -1,27 +1,8 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
```
Comment (FrederickMih) [inline src/App.js]: - [Optional] Hi @emyrue, please let's make this component class-based to fulfill the requirements of this project.

**Review by FrederickMih — APPROVED** [review]
# Status: Approved 🎉 🎉 

Hi @emyrue , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**